lastuser_core/utils.py

```diff
@@ -33,7 +33,6 @@ def make_redirect_url(url, use_fragment=False, **params):
         urlparts[3] = make_query_string(queryparts)
     return urlparse.urlunsplit(urlparts)
 
-
```

Comment:
Top level functions and classes should have two blank lines separating them. Restore this.